### DISCUSSION TITLE
[MapperIgnore] not working?

### DISCUSSION THREAD

**Discussion by yonidagan** [riok/mapperly#1252] [Q&A]
applying _[MapperIgnore]_ attribute both on the source object and the DTO, the property is still mapped both directions.
Am I missing something in the use of this attribute?

**Comment by latonz** [comment]
Thats strange, unfortunately I cannot reproduce this. Can you share a minimal reproducable?
See also `IgnoreAttributeTest` for an example: 
https://github.com/riok/mapperly/blob/e8363bf71feeba6e19a856c5f37519c26899fafa/test/Riok.Mapperly.Tests/Mapping/IgnoreAttributeTest.cs#L7-L29

**Comment by yonidagan** [comment]
Attached is a repro solution. I've figured out that the problem occurs when the mapped classes are on different projects.
(If the mapper and both mapped classes are at the same project - MapperIgnore does work as expected).

The repro solution is also available at https://github.com/yonidagan/MapperlyTest
[MapperlyTest.zip](https://github.com/riok/mapperly/files/15164941/MapperlyTest.zip)

**Reply by latonz** [reply]
The problem you are facing is that the `MapperIgnore` attribute is not present in the compiled DLL and therefore Mapperly cannot see it. To preserve the attribute in the compiled DLL you need to set the MSBuild property `MapperlyAbstractionsScope` to `runtime`. See also the [docs](https://mapperly.riok.app/docs/getting-started/installation/#preserving-the-attributes-at-runtime).
